recipes/openmesh/build.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+if test `uname` = "Darwin"
+then
+  SO_EXT='.dylib'
+
+  export CC=clang
+  export CXX=clang++
+  export MACOSX_VERSION_MIN="10.7"
+  export CXXFLAGS="${CXXFLAGS} -mmacosx-version-min=${MACOSX_VERSION_MIN}"
+  export CXXFLAGS="${CXXFLAGS} -stdlib=libc++ -std=c++11"
+  export LDFLAGS="${LDFLAGS} -mmacosx-version-min=${MACOSX_VERSION_MIN}"
+  export LDFLAGS="${LDFLAGS} -stdlib=libc++ -std=c++11"
+  export LINKFLAGS="${LDFLAGS}"
+  export MACOSX_DEPLOYMENT_TARGET=10.7
+else
+  SO_EXT='.so'
+fi
+
+mkdir -p build && cd build
+
+cmake \
+  -DCMAKE_FIND_ROOT_PATH=${PREFIX} \
+  -DCMAKE_INSTALL_PREFIX=${PREFIX} \
+  -DBUILD_APPS=OFF \
+  -DOPENMESH_PYTHON_VERSION=${PY_VER} \
+  -DOPENMESH_BUILD_PYTHON_UNIT_TESTS=ON \
+  ..
+
+make install -j${CPU_COUNT} VERBOSE=1
```

Comment:
In the long run, we are looking at overriding `CPU_COUNT` behind the scenes to ensure it is the right value. See the discussion beginning with this [comment]( https://github.com/conda-forge/conda-forge-build-setup-feedstock/pull/18#issuecomment-232986889 ) in PR ( https://github.com/conda-forge/conda-forge-build-setup-feedstock/pull/18 ). However, that is going to require another `conda-build` release before we see this happen. ( https://github.com/conda/conda-build/pull/1149 )

In the meantime, @patricksnape's suggestion is the right one. Though it is worth noting we do set `MAKEFLAGS` with `-j` specifying the right number of cores to parallelize over. So adding it directly as is done here is currently not needed.